EdvClient.js

```diff
@@ -347,41 +347,45 @@ export class EdvClient {
    * Deletes a document from the EDV.
    *
    * @param {Object} options - The options to use.
-   * @param {string} options.id the ID of the document to delete.
+   * @param {string} options.doc the document to delete.
```

Comment:
I see now that this lib does not have the jsdoc linting setup.  I think we should deal with that at a later date in a separate PR.  However, please compare these new descriptions with the existing one on L355.  The linter is going to want the dash followed by a complete sentence with proper punctuation.  Please update all new docs to conform to that format.

Reply:
I have updated the jsdoc comments to fit the correct format.